requester/requester.go

```diff
@@ -29,9 +29,39 @@ import (
 	"sync"
 	"time"
 
+	"github.com/vkuznet/x509proxy"
 	"golang.org/x/net/http2"
 )
 
+// Return client X509 certificates
+func Certs() (tls_certs []tls.Certificate) {
+	uproxy := os.Getenv("X509_USER_PROXY")
+	uckey := os.Getenv("X509_USER_KEY")
+	ucert := os.Getenv("X509_USER_CERT")
```

Comment:
Return as soon as you don't have these values set and remove the big if else block bottom.

if uproxy == "" || ukey == "" {
     return nil, nil
}